### ISSUE TITLE
Tensorboard logging image WITH LABEL

### ISSUE THREAD

**Issue by jinyu121** [pytorch/pytorch#27030]
## 🚀 Feature

We need a function that
- When log image to tensorboard, put text/label beside/on the image.
- When saving image, put text/label beside/on the image.

## Motivation

Now, we can use tensorboard to log images in PyTorch. However, we can only log images. We don't know what's the label, and/or the prediction of the image for visual debugging.
So, can you add the label to the logging image? We need a function like  `tf_put_text`, as described [HERE](https://stackoverflow.com/questions/36015170/how-can-i-add-labels-to-tensorboard-images).

## Pitch

For example:

```
|--------------------|
| class              |
|                    |
|    The image       |
|                    |
|                    |
|--------------------|
```

AND

```
|--------------------|
|                    |
|                    |
|    The image       |
|                    |
|                    |
|--------------------|
Class: 42
```

## Alternatives

Refer: `tf_put_text` function in TensorFlow

## Additional context

None

**Comment by lanpa** [comment]
https://github.com/pytorch/pytorch/pull/30941

**Comment by jinyu121** [comment]
@lanpa Thank You！

But I think #30941 is designed for detection tasks. But what should I do if I want to save the image?

For example:

Now, we can merge and save multiple images using `torchvision.utils.save_image`. The result is like this:

```
|--------------------|--------------------|--------------------|
|                    |                    |                    |
|                    |                    |                    |
|    The image       |    The image       |    The image       |
|                    |                    |                    |
|                    |                    |                    |
|--------------------|--------------------|--------------------|
```

So, how can we get a image like this:

```
|--------------------|--------------------|--------------------|
| Class:42           | Class:42           | Class:42           |
|                    |                    |                    |
|    The image       |    The image       |    The image       |
|                    |                    |                    |
|                    |                    |                    |
|--------------------|--------------------|--------------------|
```

Or, like this
```
|--------------------|--------------------|--------------------|
|                    |                    |                    |
|                    |                    |                    |
|    The image       |    The image       |    The image       |
|                    |                    |                    |
|                    |                    |                    |
|--------------------|--------------------|--------------------|
     Class: 42          Class: 42          Class: 42     

```

**Comment by Shridhar19110101** [comment]
Does this problem get solved?
